process/tsc_faq.md

```diff
@@ -0,0 +1,50 @@
+# Technical Steering Committee (TSC) FAQ
+
+The Technical Steering Committee (TSC) is the leadership of the project. This committee's primary role is to:
+
+- Set the overall direction of the project
+- Ensure the project community has the needed resources and infrastructure to succeed
+- Resolve any issues within the project community
+- Provide project updates to the TAC and the community at large
+
+Typically the TSC is comprised of the primary committers or maintainers in a project. To start there may be an appointed list, but over time more individuals may join the TSC according to the policies the TSC set for how new members are added. Regardless, TSC meetings are generally open to anyone interested unless there are sensitive issues to discuss which necessitate a private meeting.
+
+## TSC Roles
+
+Generally TSCs have a few roles to faciliate the operations of the TSC. Roles are elected by TSC voting members and term lengths are determined by the TSC.
+
+### Chairperson
+
+The TSC Chairperson is the figure head of the project. The TSC Chairperson's primary role is to:
+
+- Lead all meetings of the TSC, setting the agenda with consultation of other TSC members.
+- Be the public spokesperson for the project at events and in public communications ( such as PR/AR meetings, articles, blog posts )
+- Serve as a representative of the project to the TAC and other projects and entities.
+
+### Secretary
+
+The Secretary records all meeting notes for TSC members and ensures they are distributed to the project community after the meeting in a timely manner. Often projects have a rotating secretary role instead of a permanent role.
+
+## TSC Meetings
+
+TSC meetings are opportunities for the project community to come together to share updates and discuss issues and future plans. All TSC meetings are considered open to the public unless there is a sensitive issue to discuss ( Linux Foundation staff should be present on such meetings ).
+
+It's recommend to schedule meetings on a regular cadence ( for example, every other week at 2pm US Pacific Time ). Many project communities alternate TSC meeting times if project members are globally dispursed ( for example, one time that is US Eastern/Europe friendly and another that is US Pacific/Asia/Australia friendly ).
+
+### Scheduling logistics
+
+Each project has access to a Zoom conference account and the ability to leverage groups.io for meeting scheduling. It's recommended that the TSC Chairperson send a meeting invite with each TSC voting member as an attendee, adding the general project discussion/developer/user list ( for example, otio-discussion@lists.aswf.io ) which will automaticaly add the meeting to the calendar for those on the project email lists.
```

Comment:
Also invite `tac@lists.aswf.io`, right?

Reply:
That's a good question - that was the original guidance but I also want to be conscious of not flooding calendars. Would you think that should be the best practice going forward?

Reply:
Flooding calendars is one concern; it also makes admin'ing the events harder, because (on our google cal events at least) some of the folks subscribed to the public TAC list end up responding yes/no/maybe to the invite, and then your attendee list ends up with a collection of random folks on it. It's not exactly a problem but it makes it difficult to identify the actual TSC members.

So, I would be fine with dropping that guideline.

Maybe there's some better way of creating the events that isn't linked so strongly with email, because it is nice to see the full slate of events across the community.

Reply:
Agreed - depending upon your email list subscription this calendar serves that purpose...

https://lists.aswf.io/calendar